app/validators/embargo_date_validator.rb

```diff
@@ -6,6 +6,27 @@ class EmbargoDateValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     return if value.nil?
 
+    collection = Collection.where(id: record.model[:work][:collection_id]).first
```

Comment:
Not sure if this is the best way to retrieve the Collection associated with the work version.

Reply:
You ought to be able to do `collection = record.model.fetch(:work).collection`

Reply:
Thanks! Now updated.